src/core/caniuse.js

```diff
@@ -133,18 +141,43 @@ function validateBrowsers({ caniuse }) {
 }
 
 async function processJson(json, { feature }) {
+  /** @type {Array} */
   const results = json.result;
-  const out = results.map(({ browser: browserId, version, caniuse }) => {
-    const { name } = BROWSERS.get(browserId);
+  const groups = new Map([
+    ["desktop", []],
+    ["mobile", []],
+  ]);
+  const toBrowserCell = browserCellRenderer(feature);
+  results.reduce(toBrowserCell, groups);
+  const out = [...groups.keys()]
+    .filter(key => groups.get(key).length)
+    .map(
+      key =>
+        html`<div class="${`caniuse-group`}">
+          <div class="caniuse-browsers">${groups.get(key)}</div>
+          <div class="caniuse-type"><span>${key}</div>
```

Comment:
Not sure how this works for screen reader users. Should this be like a section heading? (We can move position with CSS)

Reply:
I think it's maybe "labelled-by"?